### ISSUE TITLE
Unix-style slash delimiters hard-coded into file paths

### ISSUE THREAD

**Issue by michaelbrooks** [CSSLint/csslint#313]
I was having issues processing the checkstyle xml output from csslint, though similar output from other tools was working -- it turned out the difference was that csslint is using unix-style slash delimiters, even on Windows. An example from my output:

``` xml
<file name="C:\path\to\src\css/style.css">
    <error line="2" column="1" severity="warning" message="@import prevents parallel downloads, use &lt;link&gt; instead." source="net.csslint.Disallow@import"/>
</file>
```

Looks like the `getFiles()` function in cli.js is hard-coded to use a "/" in the file path. The first portion of the file path is coming from Ant, which knows to use "\" on Windows. I don't know what is the right way to do this in a Node.js environment, but I know that [https://github.com/jshint/node-jshint](jshint/node-jshint) manages to do it somehow.
